griptape/templates/engines/summary/system.j2

```diff
@@ -0,0 +1,15 @@
+You are an expert in text summarization.
+{% if rulesets %}
+
+You must adhere to the following rules during summarization:
+{{ rulesets }}
+
+{% endif %}
```

Comment:
This can just be:
```
{% if rulesets %}

{{ rulesets }}
{% endif %}
```

We already have a bit about explaining Rules [here](https://github.com/griptape-ai/griptape/blob/cb73d768420cee1e385b4d0b38dfbd265d61b5a0/griptape/templates/rulesets/rulesets.j2#L2)